### PR TITLE
Add Deletion Day

### PR DESCRIPTION
**Pull request by deletionday**
- [x] I have referred the list and did not find this submission already added.
- [x] I have read the contribution guidelines [here](https://github.com/humanetech-community/awesome-humane-tech/blob/main/contributing.md)
- [x] I have read the code of conduct and I agree to adhere to the same [here](https://github.com/humanetech-community/awesome-humane-tech/blob/main/code-of-conduct.md)
- [x] I have added my entry as the latest (last dotted point) under a topic, not at the beginning, not interleaved. 

If your suggested entry is code project:

- [x] I have checked that the project is open source and has a [FSF-approved](https://www.gnu.org/licenses/license-list.html) license.

Deletion Day is [licensed under the GNU GPL v3](https://github.com/deletionday/site/blob/master/COPYING)

### Name of my software/tool/service/technology/project:

[**Deletion Day**](https://deletionday.com)

### Explain why it's a Humane Technology

Deletion Day directly opposes the privacy and security overreaches perpetrated by today’s tech industry.

### Sourcecode is available…

[…on Github](https://github.com/deletionday/site/)

### Is there a similar software/tool etc., that already exists?

I’ve found instances of one-off data deletion events, for instance:

- [The International Association of Privacy Professionals Deletion Day (2018)](https://iapp.org/news/a/data-deletion-day/)
- [Montgomery Public Schools’ “Data Deletion Week”](https://www.theguardian.com/education/2019/dec/05/schools-monitor-students-online-activity)

…but as far as I know, this is the first effort to form a recurring awareness campaign around this issue.

### Motivations, goals, and purpose

Excerpt from [deletionday.com](https://deletionday.com):

> New technologies have the power to reshape civilization. Transformative developments like the calendar and the printing press not only reorganized society; they fundamentally changed how individuals thought. Today, digital storage is cheap and abundant. As it pervades everyday life, it has shifted our expectations about permanence. In this age of internet-connectedness and cheap digital storage, we no longer need to forget.
>
> We encode our heart rates, travel plans, and friendships into permanent storage. We welcome each minor convenience, and assume that sharing and storing are positive forces – or at least, that it may come in handy later.
>
> It’s worth considering who this benefits.
>
> Industries exist with the sole purpose of scraping photos and information about you to store, analyze, and sell. Advertisers, insurance companies, and police departments profile you based on your browsing history. Photos, video, and audio recordings can be used to generate realistic “deepfakes”. Doxing and harassment campaigns are on the rise.
>
> The data you store online does not belong to you.
>
> You may be aware of this reality from the news, or just a feeling in your gut. But it can be hard to know how to respond when so much of our attention and identity are wrapped up in the Internet.
>
> On the topic of identity, it’s worth asking if uploading this much of ourselves to an unsympathetic, persistent medium was ever a good idea. As people, we grow and change. Maintaining a consistent personal brand may hamper our ability to grow as people; digital footprints don’t fade.
>
> Deletion Day challenges the prevailing notion that “more is more”. In a culture that strives for permanence, we celebrate ephemerality, growth, and change. On April 4th, a chorus of Delete keys will ring out across the world. We hope you’ll join us.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,6 +346,7 @@ Deaf and Hard of Hearing accessibility resources:
...
 - [Tracking Exposed](https://tracking.exposed/) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/tracking-exposed) - Research, browser addons on tracking, profiling, and influence of algorithms.
 - [Social Media Awareness Toolkit](https://www.smat-app.com/about) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/gitlab.svg?sanitize=true" width="16"/>](https://gitlab.com/smat-project) - An open and intuitive way to scrutinize what’s trending on internet platforms.
+- [Deletion Day](https://deletionday.com) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/deletionday/site) – A holiday to raise awareness of the myriad privacy concerns of today’s Internet. Deletion Day takes place on April 4th every year.
```
Comment (aschrijver) [inline README.md:349]: Hi @deletionday .. though you do not _really_ provide an open source software project here (other than the website code), I really like the idea this is based on.

There's one requirement however, and that is that each list entry fits on a single line (at larger resolutions). Could you change your description to _"A holiday on April 4th to raise awareness of the myriad privacy concerns of today’s Internet."_? That would fit.

Also a normal dash `-` must be used, not the longer version (forgot the char name).

Comment (aschrijver) [reply README.md:349]: PS. I don't know if this is your intention, but on Firefox most of the site title 'Deletion Day' is cut off i.e. above the site top.

Comment (deletionday) [reply README.md]: Thanks for the review, @aschrijver! I just updated the description re: your feedback.

> PS. I don't know if this is your intention, but on Firefox most of the site title 'Deletion Day' is cut off i.e. above the site top.

The site title should have a single line cut off, followed by a visible line that’s being deleted:

image

(Unless your OS “Reduce Motion” setting is turned on, in which case nothing should be cut off)

Thanks again!

Comment (aschrijver) [reply README.md]: Ah, then it is alright. This is the way it looked in my browser. Thanks for making the change. I will merge the PR.

**Comment by aschrijver** [comment]
You are now eligible to wear the humanetech badge on your site repo:

[Awesome Humane Tech](https://github.com/humanetech-community/awesome-humane-tech)

By adding this to the README:

```markdown
[![Awesome Humane Tech](https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/humane-tech-badge.svg?sanitize=true)](https://github.com/humanetech-community/awesome-humane-tech)
```

**Comment by deletionday** [comment]
Awesome ;)

Thanks!

**Pull request merged**